slidez-fe-core/src/pages/dashboard/Dashboard.tsx

```diff
@@ -83,6 +86,32 @@ const Dashboard = () => {
         })
     }
 
+    useEffect(() => {
+        setFilteredPresentations(
+            MOCK_DATA.filter((presentation) => {
+                return presentation.name
+                    .toLowerCase()
+                    .includes(searchField.toLowerCase())
+            })
+        )
```

Comment:
You can use it without return statement
MOCK_DATA.filter((presentation) => presentation.name
                    .toLowerCase()
                    .includes(searchField.toLowerCase())
            )